MekHQ/src/mekhq/campaign/rating/FieldManualMercRevDragoonsRating.java

```diff
@@ -520,36 +520,19 @@ public int getCommanderValue() {
             return 0;
         }
 
-        int value = 0;
-
-        Skill test = getCommander().getSkill(SkillType.S_LEADER);
-        if (test != null) {
-            value += test.getLevel();
-        }
-
-        test = getCommander().getSkill(SkillType.S_NEG);
-        if (test != null) {
-            value += test.getLevel();
-        }
-
-        test = getCommander().getSkill(SkillType.S_STRATEGY);
-        if (test != null) {
-            value += test.getLevel();
-        }
-
-        test = getCommander().getSkill(SkillType.S_TACTICS);
-        if (test != null) {
-            value += test.getLevel();
-        }
+        int skillTotal = getCommanderSkillLevelWithBonus(SkillType.S_LEADER);
```

Comment:
Reading FM:Mr(Rev) one more time, after reviewing CamOps, I believe this needs to change but CamOps is fine.

Relevant text (pp153):
> - Commander Rating Value
> - Positive Trait (each) +1
> - Negative Trait (each) –1
> - Leadership Skill As skill bonus
> - Negotiations Skill As skill bonus
 >- Strategy Skill As skill bonus
> - Tactics Skill As skill bonus

Searching for a table which talks about skill bonus, I find (pp160):
> NEGOTIATION MODIFIERS TABLE
> Negotiator Rating    Skill Bonus    Freelance Fee
> Green     +1     1,800
> Regular     +3     2,400
> Veteran     +5     4,800
> Elite     +7     7,200

Elsewhere talks about leadership skill bonus of '+3'. I believe this means for Dragoon's Rating we should be using this custom bonus rather than the direct skill level plus bonus.